hebiros_gazebo_plugin/include/hebiros_gazebo_controller.cpp

```diff
@@ -178,6 +184,105 @@ void HebirosGazeboController::SetDefaultGains(std::shared_ptr<HebirosGazeboGroup
     hebiros_group->settings.effort_gains.ki.push_back(0);
     hebiros_group->settings.effort_gains.kd.push_back(0.001);
   }
+  if (model_name == "X8_3" && control_strategy == 2) {
+    hebiros_group->settings.position_gains.kp.push_back(3);
+    hebiros_group->settings.position_gains.ki.push_back(0);
+    hebiros_group->settings.position_gains.kd.push_back(0);
+    hebiros_group->settings.velocity_gains.kp.push_back(0.1);
+    hebiros_group->settings.velocity_gains.ki.push_back(0);
+    hebiros_group->settings.velocity_gains.kd.push_back(0);
+    hebiros_group->settings.effort_gains.kp.push_back(0.1);
+    hebiros_group->settings.effort_gains.ki.push_back(0);
+    hebiros_group->settings.effort_gains.kd.push_back(0.0001);
+  }
+  else if (model_name == "X8_3" && control_strategy == 3) {
+    hebiros_group->settings.position_gains.kp.push_back(1);
+    hebiros_group->settings.position_gains.ki.push_back(0);
```

Comment:
This way is so much more bug prone than if you did something like

if (model_name == "X8_3") {
  hebiros_group->settings.position_gains.ki.push_back(0);
  hebiros_group->settings.position_gains.kd.push_back(0);
  // rest of gains values which are all the same, regardless of control_strategy
  if (control_strategy == 2) {
    // strat 2 dependent gains
  } // etc
}

Reply:
Originally I had done things like that; I've switched to this method because when we change a number, we often just change it for one control strategy.  This takes more lines, but matches up with downloadable "default gains" _much_ easier, and ends up being less error prone.